### ISSUE TITLE
Cannot add Percentage as a column

### ISSUE THREAD

**Issue by Kuuchuu** [nocodb/nocodb#1129]
NocoDB used as docker : **true**
NocoDB version : **0.84.9**
Database used in NC_DB URL  : **pg**
Project was created  by clicking : **New Project by connecting to external database**
Database on which spreadsheet is created : **pg**
OS on which NocoDB is running : **Docker/Rocky Linux 8**
Database version : **14.1-1.pgdg110+1**
Node : **v12.22.6**
Arch : **x64**
ProjectOnRootDB: **true**

**Steps To Reproduce**
1. Go to a table
2. Add new column --> Percentage type
3. "Update table failed", no column is added.

**Expected behavior**
Should add Percentage column.

Dev Console Error:
```
POST https://noco.anchorhrg.com/dashboard/?q=sqlOpPlus_tableUpdate 500
(anonymous) @ 3370c21.js:2
t.exports @ 3370c21.js:2
t.exports @ 3370c21.js:2
Promise.then (async)
v.request @ 3370c21.js:2
(anonymous) @ 3370c21.js:2
(anonymous) @ 35a1cff.js:2
m @ 5d83827.js:2
(anonymous) @ 5d83827.js:2
A.forEach.t.<computed> @ 5d83827.js:2
r @ 3370c21.js:2
l @ 3370c21.js:2
(anonymous) @ 3370c21.js:2
(anonymous) @ 3370c21.js:2
ActSqlOpPlus @ 35a1cff.js:2
(anonymous) @ 3370c21.js:2
m.dispatch @ 3370c21.js:2
dispatch @ 3370c21.js:2
(anonymous) @ 99706d2.js:2
m @ 5d83827.js:2
(anonymous) @ 5d83827.js:2
A.forEach.t.<computed> @ 5d83827.js:2
r @ 3370c21.js:2
l @ 3370c21.js:2
(anonymous) @ 3370c21.js:2
(anonymous) @ 3370c21.js:2
save @ 99706d2.js:2
ee @ 3370c21.js:2
n @ 3370c21.js:2
ee @ 3370c21.js:2
t.$emit @ 3370c21.js:2
click @ 5d83827.js:2
ee @ 3370c21.js:2
n @ 3370c21.js:2
Fr.c._wrapper @ 3370c21.js:2
35a1cff.js:2 index 0
99706d2.js:2 Error: Request failed with status code 500
    at t.exports (3370c21.js:2:178381)
    at t.exports (3370c21.js:2:208481)
    at XMLHttpRequest.A (3370c21.js:2:176975)
```

**Comment by o1lab** [comment]
```
Within DB_TABLE_UPDATE handler {
  api: 'tableUpdate',
  env: '_noco',
  dbAlias: 'db',
  project_id: 'neat_dingo_f4a6',
  args: {
    tn: 'a',
    _tn: 'a',
    originalColumns: [ [Object], [Object], [Object], [Object] ],
    columns: [ [Object], [Object], [Object], [Object], [Object] ]
  },
  sqlOpPlus: true
}
{
  api: 'tableUpdate',
  env: '_noco',
  dbAlias: 'db',
  project_id: 'neat_dingo_f4a6',
  args: {
    tn: 'a',
    _tn: 'a',
    originalColumns: [ [Object], [Object], [Object], [Object] ],
    columns: [ [Object], [Object], [Object], [Object], [Object] ]
  },
  sqlOpPlus: true
}
╔═════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════╗
║                                                                                                                                 ║
║                                                 EXCEPTION OCCURED!! in PGClient @ tableUpdate                                   ║
║   - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -           ║
║   MESSAGE:                                                                                                                      ║
║   ALTER TABLE "a"  ADD "title5" double NULL ;; - type "double" does not exist                                                   ║
║                                                                                                                                 ║
║   - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -           ║
║   CODE:                                                                                                                         ║
║   42704                                                                                                                         ║
║                                                                                                                                 ║
║   - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -           ║
║   STACK:                                                                                                                        ║
║   error: ALTER TABLE "a"  ADD "title5" double NULL ;; - type "double" does not exist                                            ║
║       at Parser.parseErrorMessage (/Users/navi/practice/nocodb/packages/nocodb/node_modules/pg-protocol/src/parser.ts:369:69)   ║
║       at Parser.handlePacket (/Users/navi/practice/nocodb/packages/nocodb/node_modules/pg-protocol/src/parser.ts:188:21)        ║
║       at Parser.parse (/Users/navi/practice/nocodb/packages/nocodb/node_modules/pg-protocol/src/parser.ts:103:30)               ║
║       at Socket.<anonymous> (/Users/navi/practice/nocodb/packages/nocodb/node_modules/pg-protocol/src/index.ts:7:48)            ║
║       at Socket.emit (node:events:390:28)                                                                                       ║
║       at Socket.emit (node:domain:475:12)                                                                                       ║
║       at addChunk (node:internal/streams/readable:324:12)                                                                       ║
║       at readableAddChunk (node:internal/streams/readable:297:9)                                                                ║
║       at Socket.Readable.push (node:internal/streams/readable:234:10)                                                           ║
║       at TCP.onStreamRead (node:internal/stream_base_commons:190:23)                                                            ║
║                                                                                                                                 ║
║   - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -           ║
║                                                                                                                                 ║
║                                                                                                                                 ║
╚═════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════╝
error: ALTER TABLE "a"  ADD "title5" double NULL ;; - type "double" does not exist
    at Parser.parseErrorMessage (/Users/navi/practice/nocodb/packages/nocodb/node_modules/pg-protocol/src/parser.ts:369:69)
    at Parser.handlePacket (/Users/navi/practice/nocodb/packages/nocodb/node_modules/pg-protocol/src/parser.ts:188:21)
    at Parser.parse (/Users/navi/practice/nocodb/packages/nocodb/node_modules/pg-protocol/src/parser.ts:103:30)
    at Socket.<anonymous> (/Users/navi/practice/nocodb/packages/nocodb/node_modules/pg-protocol/src/index.ts:7:48)
    at Socket.emit (node:events:390:28)
    at Socket.emit (node:domain:475:12)
    at addChunk (node:internal/streams/readable:324:12)
    at readableAddChunk (node:internal/streams/readable:297:9)
    at Socket.Readable.push (node:internal/streams/readable:234:10)
    at TCP.onStreamRead (node:internal/stream_base_commons:190:23) {
  length: 93,
  severity: 'ERROR',
  code: '42704',
  detail: undefined,
  hint: undefined,
  position: '31',
  internalPosition: undefined,
  internalQuery: undefined,
  where: undefined,
  schema: undefined,
  table: undefined,
  column: undefined,
  dataType: undefined,
  constraint: undefined,
  file: 'parse_type.c',
  line: '274',
  routine: 'typenameType'
}
error: ALTER TABLE "a"  ADD "title5" double NULL ;; - type "double" does not exist
    at Parser.parseErrorMessage (/Users/navi/practice/nocodb/packages/nocodb/node_modules/pg-protocol/src/parser.ts:369:69)
    at Parser.handlePacket (/Users/navi/practice/nocodb/packages/nocodb/node_modules/pg-protocol/src/parser.ts:188:21)
    at Parser.parse (/Users/navi/practice/nocodb/packages/nocodb/node_modules/pg-protocol/src/parser.ts:103:30)
    at Socket.<anonymous> (/Users/navi/practice/nocodb/packages/nocodb/node_modules/pg-protocol/src/index.ts:7:48)
    at Socket.emit (node:events:390:28)
    at Socket.emit (node:domain:475:12)
    at addChunk (node:internal/streams/readable:324:12)
    at readableAddChunk (node:internal/streams/readable:297:9)
    at Socket.Readable.push (node:internal/streams/readable:234:10)
    at TCP.onStreamRead (node:internal/stream_base_commons:190:23) {
  length: 93,
  severity: 'ERROR',
  code: '42704',
  detail: undefined,
  hint: undefined,
  position: '31',
  internalPosition: undefined,
  internalQuery: undefined,
  where: undefined,
  schema: undefined,
  table: undefined,
  column: undefined,
  dataType: undefined,
  constraint: undefined,
  file: 'parse_type.c',
  line: '274',
  routine: 'typenameType'
}

```

**Comment by wingkwong** [comment]
Fixed in ``0.84.10``.
